code/datums/mutations/speech.dm

```diff
@@ -220,15 +220,87 @@
 	owner.grant_language(/datum/language/common)
 	owner.remove_language(/datum/language/beachbum)
 
+/datum/mutation/human/shouting
+	name = "Shouting"
+	desc = "A mutation that forces the host to constantly shout their sentences out."
+	quality = MINOR_NEGATIVE
+	locked = TRUE
+	text_gain_indication = "<span class='warning'>You feel angry.</span>"
+	text_lose_indication = "<span class='notice'>You feel calm.</span>"
+
+/datum/mutation/human/shouting/say_mod(message)
+	if(message)
+		message = "[uppertext(replacetext(message, ".", "!"))]!"
+	return (message)
+
 /datum/mutation/human/yelling
 	name = "Yelling"
 	desc = "A mutation that forces the host to constantly yell their sentences out."
 	quality = MINOR_NEGATIVE
 	locked = TRUE
-	text_gain_indication = "<span class='danger'>You feel really angry.</span>"
-	text_lose_indication = "<span class='notice'>You feel calmer.</span>"
+	text_gain_indication = "<span class='danger'>You feel REALLY angry.</span>"
+	text_lose_indication = "<span class='notice'>You feel calm.</span>"
 
 /datum/mutation/human/yelling/say_mod(message)
 	if(message)
-		message = "[uppertext(replacetext(message, ".", "!"))]!"
+		message = "[uppertext(replacetext(message, ".", "!"))]!!"
 	return (message)
+
+/datum/mutation/human/canadian
+	name = "Canadian"
+	desc = "Unknown"
+	quality = MINOR_NEGATIVE
+	text_gain_indication = "<span class='notice'>You feel polite.</span>"
+	text_lose_indication = "<span class='danger'>You feel american.</span>"
+
+/datum/mutation/human/canadian/say_mod(message)
```

Comment:
why is this a mutation and not a trait?

Reply:
i made it a mutation since other speech modifications were mutations so i saw it fit to put it there, but i can change them to traits.

Reply:
I'm sorry if this ruins vilebeggar's life but don't we already have functionality for clothing to dictate say? IE lucha mask, beret, etc.

Reply:
https://github.com/tgstation/tgstation/blob/ccadcf61d3d383191d950a8ac5a350b0213ba9e9/code/modules/clothing/masks/miscellaneous.dm#L47

https://github.com/tgstation/tgstation/blob/1b670a9c36abdf840f1ea54058df8a607dbdea7c/code/modules/clothing/head/misc.dm#L353

Reply:
rip

Reply:
oh dude s-word you cobby i'll have to shitcode this in tomorrow

Reply:
see the format for those as well in turning your canada speak into a json file.